charts/opentelemetry-operator/templates/rolebinding.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.role.create }}
+{{- if .Values.manager.leaderElection.enabled }}
```

Comment:
Can we combine this if statement with the one above?

Reply:
done